cmd/controller-manager/app/controllermanager.go

```diff
@@ -3,6 +3,7 @@ package app
 import (
 	"context"
 	"flag"
+	"k8s.io/klog/v2"
```

Comment:
This line maybe needs to be put into the second import block.

Reply:
oops, didn't notice why that line was changed, fixed